react/util/ui-utils.js

```diff
@@ -98,19 +98,21 @@ export const pickImageCamera = async function ({
 
 export const pickImageGalery = async function ({
     mediaType = "photo",
+    selectionLimit = 5,

```

Comment:
Is this required?

Reply:
@gcandal yes, but maybe I should set to 1
<img width="760" alt="ertyup" src="https://user-images.githubusercontent.com/8842023/121920810-4da95780-cd30-11eb-9dee-fe02062e3c70.png">

Reply:
Go with 0 then